tests/test_trajectory.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+import unittest
+from ddt import ddt, data, unpack
+
+import numpy as np
+
+import broni
+from broni.shapes.primitives import Cuboid, Sphere
+
+
+@ddt
+class TestTrajectory(unittest.TestCase):
+    def setUp(self):
```

Comment:
probably not needed to define setUp and tearDown if they do nothing?

Reply:
I tend to add them also, but I agree this adds some noise.

Reply:
Copied from the best.

Reply:
removed, though.